packages/augur-ui/src/modules/markets-list/containers/markets-view-container.ts

```diff
@@ -142,6 +145,11 @@ const mapDispatchToProps = (
         pageNumber
       )
     ),
+  hotLoadMarketList: async (cb) => {
+    const marketsInfo = await augurSdkLite.get().getMarketCreatedLogs('0x919274CE3Ba5a0D0f11071Bb9fb402AE5c23381B');
```

Comment:
how come you put a hardcoded address in here? testing?

Reply:
Good catch! Fixed